config/servicemappings/alloydb.yaml

```diff
@@ -105,7 +105,6 @@ spec:
         - tfField: network
           description: |-
             Only `external` field is supported to configure the reference.
-            
             (Required) The relative resource name of the VPC network on which 
             the instance can be accessed. It is specified in the following form: 
             projects/{projectNumber}/global/networks/{network_id}."
```

Comment:
I'm guessing the weird format in the generated CRD is caused by the additional double quote in the end of this sentence. Could you try to remove it?

Reply:
Done. Good catch!